README.md

```diff
@@ -35,6 +35,17 @@ Featured in the video:
 * Showing diagnostic message in a floating window.
 * Code actions.
 
+## How to get the most out of this plugin
+
+This plugin is designed to provide an opinionated and useable set of defaults for working with LSP and Completion in Neovim. A few things should be considered before you use it in your own config
+
+* To get the best experience, it is recommended to read through this page from top to bottom and to understand the options available to you. 
+* Many things that users often try to customize outside of this plugin are handled through the plugin functionality, and the "doubling up" of configuration can cause issues and loss of functionality. If you are using this plugin and want to customize anything, it is recommended to follow the instructions below and in the [Advance usage](https://github.com/VonHeikemen/lsp-zero.nvim/blob/main/advance-usage.md) page. 
+* Any setup code used outside the built in configuration options and related to LSP or Completion may result in undesired behavior. 
+* This plugin also handles many layers of abstraction in an effort to give a broad range of users the best possible experience out of the box. Some things you might be expecting might not be present as a result. If you feel something is missing, search the [discussions](https://github.com/VonHeikemen/lsp-zero.nvim/discussions) to see if it has already been discussed, and open a new discussion if not.  
+* If you want to set up completion yourself, you still can. See the [Available Presets](https://github.com/VonHeikemen/lsp-zero.nvim#available-presets) section for instructions on how to disable that portion of the plugin and assure your own setup does not conflict with the plugin
```

Comment:
I think we can link to a section in the advance usage page here. Maybe this:

https://github.com/VonHeikemen/lsp-zero.nvim/blob/main/advance-usage.md#the-current-api-is-not-enough

Probably we should improve the advance usage page too.

Reply:
> I think we can link to a section in the advance usage page here. Maybe this:

Please clarify, you mean the link on line 43 I assume?

Reply:
Line 46. The one about setting up completions.

Reply:
Understood and added.